package.json

```diff
@@ -16,38 +16,41 @@
     "packages/*"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.4.4",
-    "@babel/core": "^7.4.5",
-    "@babel/helper-validator-identifier": "^7.9.0",
-    "@babel/plugin-transform-runtime": "^7.7.6",
-    "@babel/preset-env": "^7.4.5",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.8.3",
-    "@babel/runtime": "^7.7.7",
+    "@babel/cli": "^7.10.1",
+    "@babel/core": "^7.10.2",
+    "@babel/helper-validator-identifier": "^7.10.1",
+    "@babel/plugin-transform-runtime": "^7.10.1",
+    "@babel/preset-env": "^7.10.2",
+    "@babel/preset-react": "^7.10.1",
+    "@babel/preset-typescript": "^7.10.1",
+    "@babel/runtime": "^7.10.2",
     "@testing-library/react": "^9.1.3",
     "@types/jest": "^25.1.2",
     "@types/lodash.debounce": "^4.0.6",
     "@types/lodash.merge": "^4.6.6",
     "@types/react-test-renderer": "^16.9.2",
-    "babel-jest": "^25.3.0",
+    "babel-jest": "^26.0.1",
     "husky": ">=4.0.7",
-    "jest": "^24.8.0",
+    "jest": "^26.0.1",
     "jest-canvas-mock": "^2.2.0",
-    "jest-emotion": "^10.0.11",
-    "jest-mock-console": "^1.0.0",
+    "jest-emotion": "^10.0.32",
+    "jest-mock-console": "^1.0.1",
     "lerna": "^3.14.1",
     "lint-staged": "10",
     "microbundle": "^0.11.0",
     "prettier": "^2.0.5",
     "react-test-renderer": "^16.8.6",
-    "ts-jest": "^25.2.0"
+    "ts-jest": "^26.1.0",
+    "ts-snippet": "^4.2.0",
+    "typescript": "^3.9.5"
   },
   "resolutions": {},
   "jest": {
     "preset": "ts-jest/presets/js-with-babel",
     "globals": {
       "ts-jest": {
         "tsConfig": {
+          "target": "es2016",
```

Comment:
I bumped Babel and Jest deps to use optional chaining in tests.

- [ ] The target here could be increased to `es2018` I guess.